### PR TITLE
 Replace will_paginate with kaminari 

### PR DESCRIPTION
**Pull request by sonalkr132**
COUNT(DISTINCT ..) used in pagination for page number links is very expensive
in this case. This change will only show `next` and `prev` page link.

response time of /gems/rails/reverse_dependencies (tested locally)
before: 0m1.564s
after: 0m0.708s

Related: #1798

### CODE REVIEW

**Comment by dwradcliffe** [comment]
🤣 I guess this is super approved since I tried to approve during the GitHub outage...

**Comment by sonalkr132** [comment]
@simi updated!

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -51,4 +51,8 @@ def search_form_class
   def active?(path)
     "is-active" if request.path_info == path
   end
+
+  def without_count_paginate(items)
```
Comment (simi) [inline app/helpers/application_helper.rb]: I think `paginate_without_count` looks better. But both describes actual implementation detail and not the expected output of this helper (having pagination missing numbered steps -> prev/next links only). What about some more generic name? Maybe `simple_paginate` (and leave a comment about performance in here)?

**Comment by sonalkr132** [comment]
>  What about some more generic name? Maybe simple_paginate (and leave a comment about performance in here)?

Used plain_paginate and added comment.
> plain
/pleɪn/Submit
adjective
>
> 1. not decorated or elaborate; simple or basic in character.
"good plain food"
synonyms: | simple, ordinary, unadorned, undecorated, unembellished

pun-dog-seo-expert

**Comment by sonalkr132** [comment]
Merged locally, had to update vendor.

**Comment by simi** [comment]
Great @sonalkr132!

**Pull request closed without merging**